internal/pods/schedule.go

```diff
@@ -300,6 +300,10 @@ func addNodeSelectorTerm(nodeSelTerms []v1.NodeSelectorTerm, label string,
 }
 
 func checkNSLabels(config *Config) error {
+	if config.Namespace == constants.OperatorNamespace {
```

Comment:
Would be good to have a comment explaining the "why" here, so that future people reading the code understand why it's skipped on this namespace